build/assets/custom-dictionary.txt

```diff
@@ -245,3 +245,11 @@ bedtools
 IRanges
 mergeByOverlaps
 cytoband
+ANOVA
+polyA
```

Comment:
I just standardized this! change text to "poly-A"

Reply:
Sorry I thought it went the _other way_ and was confused.